src/bitcoind.ts

```diff
@@ -1,5 +1,146 @@
+import Client from "bitcoin-core"
 import _ from "lodash"
-import { BitcoindClient, bitcoindDefaultClient, btc2sat } from "./utils"
+import { btc2sat } from "./utils"
+
+const connection_obj = {
+  network: process.env.NETWORK,
+  username: "rpcuser",
+  password: process.env.BITCOINDRPCPASS,
+  host: process.env.BITCOINDADDR,
+  port: process.env.BITCOINDPORT,
+  version: "0.21.0",
+}
+
+type ScriptPubKey = {
+  asm: string
+  hex: string
+  reqSigs: number
+  type: string
+  addresses: [string]
+}
+
+type VOut = {
+  value: number
+  n: number
+  scriptPubKey: ScriptPubKey
+}
+
+type GetAddressInfoResult = {
+  address: string
+  // TODO
+}
+
+type InWalletTransaction = {
+  amount: number
+  fee: number
+  // TODO
+  txid: string
+}
+
+// TODO consistency in params
```

Comment:
In these newly added classes, some methods use a json object as input while others use direct parameters. Should it be consistent? Which one should we choose?

Reply:
I think we're converging on `json` as input typed via `XxxConstructorArgs`.